src/Util/MailerLoggerUtil.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Util;
+
+use Symfony\Bridge\Monolog\Logger;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+use Swift_Events_SendEvent;
+use Swift_Events_SendListener;
+
+/**
+ * A utility to log Swift e-mails.
+ */
+class MailerLoggerUtil implements Swift_Events_SendListener
+{
+     /**
+      * A Monolog logger.
+      *
+      * @var Logger
+      */
+    protected $logger;
+
+    /**
+     * MailerLoggerUtil constructor.
+     *
+     * @param LoggerInterface $logger The Logger Interface.
+     */
+    public function __construct(LoggerInterface $logger)
```

Comment:
`Psr\Log\LoggerInterface $<channel>Logger` Can be used to specify the channel for the logger and the arguments in the servers.yaml can be removed